internal/engine/engine.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/artifacts"
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/internal/check"
+	certcheck "github.com/redhat-openshift-ecosystem/openshift-preflight/internal/check"
```

Comment:
NIT: Why is this changing?

Reply:
Agreed. This is an awkward change.

Reply:
This is due to scoping, I can't reference the `check` package in a scoped for loop on line 223, since `check` is a struct there. If there is a better way to go about this LMK.